src/pages/components/split-button.mdx

```diff
@@ -39,6 +39,40 @@ components:
 
 ## How to use it
 
+### Default
+
+The typical usage of a Split Button component.
+
+import SplitButtonDefault from '../../examples/button/SplitButtonDefault.tsx';
+import SplitButtonDefaultCode from '!!raw-loader!../../examples/button/SplitButtonDefault.tsx';
+
+<CodeExample code={SplitButtonDefaultCode}>
+  <SplitButtonDefault />
+</CodeExample>
+
+### Danger
+
+Use danger styling for Split Buttons that enable destructive action.
+
+import SplitButtonDanger from '../../examples/button/SplitButtonDanger.tsx';
+import SplitButtonDangerCode from '!!raw-loader!../../examples/button/SplitButtonDanger.tsx';
+
+<CodeExample code={SplitButtonDangerCode}>
+  <SplitButtonDanger />
+</CodeExample>
+
+### Disabled
+
+A disabled Split Button prevents user interaction. It doesn’t appear in the tab order,
+can’t receive focus, and may not read aloud by a screenreader.
+
+import SplitButtonDisabled from '../../examples/button/SplitButtonDisabled.tsx';
+import SplitButtonDisabledCode from '!!raw-loader!../../examples/button/SplitButtonDisabled.tsx';
+
+<CodeExample code={SplitButtonDisabledCode}>
+  <SplitButtonDisabled />
+</CodeExample>
+
 ### Size
 
 Split Buttons can be small, default, or large.
```

Comment:
Maybe this is happening in a different PR somewhere else, but we updated these to use the word medium and say what the default is.

Reply:
Good spot. We did fix this in other places but this is new.